packages/office-desk.yaml

```diff
@@ -23,8 +24,8 @@ esphome:
       - button.press: button_m
 
 external_components:
-  source: github://iMicknl/LoctekMotion_IoT@v2.1
-  components: [ loctekmotion_desk_height_sensor ]
+  source: github://iMicknl/LoctekMotion_IoT@v2.2
+  components: [ loctekmotion_desk_height ]
```

Comment:
This feels like a better name for the component, since "sensor" seemed redundant